modules/baremetal-event-relay.adoc

```diff
@@ -6,8 +6,9 @@
 
 [discrete]
 == Purpose
-The OpenShift {redfish-operator} manages the life-cycle of the Baremetal Event Relay. The Baremetal Event Relay enables you to configure the types of cluster event that are monitored using Redfish hardware events.
+The OpenShift {redfish-operator} manages the life-cycle of the Bare Metal Event Relay. The Bare metal Event Relay enables you to configure the types of cluster event that are monitored using Redfish hardware events.
```

Comment:
Bare metal is being used as an adj, so it should be hyphenated. Also, the 2nd instance should have "metal" with leading cap.

Reply:
This is the name of the Operator and that's its official iteration, without hyphens.